### PR TITLE
1 setup django

### PR DESCRIPTION
**Pull request by lucaslootalot**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "image": "ghcr.io/naturerobots/hsos-sep-plant-map-2022:main",
+  "image": "ghcr.io/lucaslootalot/hsos-sep-plant-map-2022:main",
```
Comment (spuetz) [inline .devcontainer/devcontainer.json:2]: We will not add your local container registry to the main code base. ;)

**Review by spuetz — CHANGES_REQUESTED** [review]
@mhoellmann or @mklpiening Could we make the GitHub organization or user (naturerobots vs lucaslootalot) or the package path a variable here? Or should all have read and write access to the package?

**Comment by spuetz** [comment]
@lucaslootalot Why did you close it? Just push the changes to the branch.

**Comment by lucaslootalot** [comment]
@spuetz  I did not want to create a pull request at this repo. That's why I closed it again.

**Comment by lucaslootalot** [comment]
Closed due to unintentional pull request.

**Pull request closed without merging**